### PR TITLE
Release 7.5.3

### PR DESCRIPTION
**Pull request by crow**
Patch release for security update

### CODE REVIEW

File: CHANGELOG [hunk 1]
```diff
@@ -1,3 +1,14 @@
+=============================
+Version 7.5.3 - March 14, 2019
+=============================
+=============================
```
Comment (marc-scig) [inline CHANGELOG]: is there an extra field of `=` here?

File: CHANGELOG [hunk 2]
```diff
@@ -1,3 +1,14 @@
...
+- Fixed a security issue within Urban Airship SDK, that could allow trusted
+URL redirects in certain edge cases.
+
+All applications that are using Urban Airship SDK 9.0.0 - 9.7.0 should
```
Comment (rlepinski) [inline CHANGELOG]: You should specify cordova versions

**Comment by rlepinski** [comment]
👍

**Comment by marc-scig** [comment]
🍸

File: CHANGELOG (hunk 1 above)
Comment (crow) [reply CHANGELOG]: weird, yeah, fixed

**Pull request merged**